messaging/amqp-reactive/src/test/java/io/quarkus/ts/messaging/amqpreactive/ProdAmqpReactiveIT.java

```diff
@@ -32,7 +32,8 @@ public class ProdAmqpReactiveIT extends BaseAmqpReactiveIT {
     static RestService app = new RestService()
             .withProperty("amqp-host", amq::getAmqpHost)
             .withProperty("amqp-port", () -> "" + amq.getPort())
-            .withProperty("quarkus.otel.exporter.otlp.traces.endpoint", jaeger::getCollectorUrl);
+            .withProperty("quarkus.otel.exporter.otlp.traces.endpoint", jaeger::getCollectorUrl)
+            .withProperty("quarkus.otel.bsp.schedule.delay", "3S");
```

Comment:
how does it make sense to make interval between 2 exports 3 seconds? why should we wait? 

Export should take hundred of millis at best, in startstops we use

```
quarkus.otel.bsp.schedule.delay=700ms
quarkus.otel.bsp.export.timeout=700ms
```

Reply:
I agree that we can and should go lower than 3 seconds as we only expecting 1 trace. Probably 1 second or 500ms is enough. (as stated before I tested 1 second and it was fine without changing the rest of the code).